packages/experiments/src/components/fluent/FluentStyles.ts

```diff
@@ -53,22 +54,56 @@ const BreadcrumbStyles = {
 
 const PrimaryButtonStyles = {
   root: {
-    borderRadius: '2px',
-    boxShadow: FluentDepthLevels.Level1
+    borderRadius: '2px'
+    // boxShadow: FluentDepthLevels.Level1
```

Comment:
Maybe delete these comments?

Reply:
For now, I'd prefer to leave them.  That should be the shadow that ends up on the button, but for right now there's a border instead.  Its commented out because I expect to put it back in at some point.  I can add a comment that explains that clearly tho :)

Reply:
@oengusmacinog, can you put those lines somewhere for safe keeping? I get that we want to keep them around as they WILL come back, but I also see @dzearing's rage for order here too.